test/unit_tests/modules/util/atomic-helpers-spec.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+
+var chai = require( 'chai' );
+var assert = chai.assert;
```

Comment:
Yeah, it's mostly the syntax for how you write the tests. Expect is a style for behavior driven development, Assert is a style more for test driven development. There's a good explanation of the differences here. 

http://chaijs.com/guide/styles/